src/plugins/vis_types/vislib/public/vislib/lib/dispatch.js

```diff
@@ -375,14 +374,13 @@ export class Dispatch {
   /**
    * return function to Highlight the element that is under the cursor
    * by reducing the opacity of all the elements on the graph.
-   * @param uiSettings
    * @method getHighlighter
    */
-  getHighlighter(uiSettings) {
+  getHighlighter() {
     return function highlight(element) {
       const label = this.getAttribute('data-label');
       if (!label) return;
-      const dimming = uiSettings.get(DIMMING_OPACITY_SETTING);
+      const dimming = 0.5;
       $(element)
         .parent()
         .find('[data-label]')
```

Comment:
Nit: in line 389 `justifyOpacity` is used - it's only necessary for custom opacity values so it can be simplified further (just setting it to `0.5`)

Reply:
Fixed https://github.com/elastic/kibana/pull/111704/commits/7f079cb21cdbd43188171307a77cb999d73a8839